firebase-firestore/src/main/java/com/google/firebase/firestore/model/Document.java

```diff
@@ -66,6 +67,11 @@ public ObjectValue getData() {
     return objectValue.get(path);
   }
 
+  public @Nullable Value getFieldProto(FieldPath path) {
+    FieldValue fieldValue = getField(path);
```

Comment:
ObjectValue.getFieldProto() does not yet exist (but it probably should).

That being said, this will go away with https://github.com/firebase/firebase-android-sdk/pull/1207, which will drop the FieldValue version altogether.